packages/web/src/scss/settings/_utilities.scss

```diff
@@ -148,8 +148,6 @@ $utilities: (
             primary: tokens.$background-primary,
             secondary: tokens.$background-secondary,
             tertiary: tokens.$background-tertiary,
-            brand-primary: tokens.$background-brand-primary,
-            brand-secondary: tokens.$background-brand-secondary,
```

Comment:
Do we have this removal documented in the MG? I think it's there, but we can be specific now and tell exactly which utility classes were removed.